app/proxy/tls.ts

```diff
@@ -25,7 +25,7 @@ function setCommon(cert: pki.Certificate, commonName: string, issuer?: pki.Certi
   cert.validity.notAfter = new Date();
   // As described at https://support.apple.com/en-gb/HT210176, certificates
   // must be valid for 825 days or less.
-  cert.validity.notAfter.setFullYear(cert.validity.notBefore.getFullYear() + 2);
+  cert.validity.notAfter.setFullYear(cert.validity.notBefore.getFullYear() + 1);
```

Comment:
Could you please update the preceding comment with the new validity period and a link to the new resource (https://chromium.googlesource.com/chromium/src/+/HEAD/net/docs/certificate_lifetimes.md) that you found?

Reply:
update to pull request incoming, totally oversaw that comment